README.md

```diff
@@ -34,6 +41,66 @@ Write a function to generate the fibonacci numbers
   - Write a function, `fibPairs`, so that when given an integer `n`, `fibPairs` returns a pair of integers representing the (n - 1)th and and nth fibonnacci numbers. You can take the -1st element of the fibonnaci sequence to be 0, and the 0th to be 1.
   - Write a function `fibonnaci` that maps an integer n to the nth fibonnacci number.
 
+## Recursion
+
+Spoiler warning -- Don't look in the Data.List module until you have working answers to this chapter's exercises.
+
+### Exercise (isPrefixOf)
+```haskell
+-- | The 'isPrefixOf' function takes two lists and returns 'True'
+-- iff the first list is a prefix of the second.
+isPrefixOf              :: (Eq a) => [a] -> [a] -> Bool
+```
+* Find a **recursive** implementation of `isPrefixOf`
+* Why do we enforce the typeclass constraint `(Eq a)`?
+
+### Exercise (intersperse)
+```haskell
+-- | The 'intersperse' function takes an element and a list and
+-- \`intersperses\' that element between the elements of the list.
+-- For example,
+--
+-- > intersperse ',' "abcde" == "a,b,c,d,e"
+```
+* Implement `intersperse` recursively
+
+### Exercise (transpose)
+```haskell
+-- | The 'transpose' function transposes the rows and columns of its argument.
+-- For example,
+--
+-- > transpose [[1,2,3],[4,5,6]] == [[1,4],[2,5],[3,6]]
+```
+* Implement `transpose`recursively
+
+## Higher Order Functions
+
+### Exercise (primes)
+
+Write functions that produce:
+* The 1st n primes
+* The nth prime
+* The first prime above a given integer
+
+Attempt the functions after reading **Maps and Filters**.
```

Comment:
Do you mean the prime functions above?

Reply:
Ah yeah, I'll make that a bit more clear